src/classes/trackers.js

```diff
@@ -78,6 +78,10 @@
  */
 
 class Trackers {
+    static supportedRuleActions = new Set([
+        'ad-attribution', 'block', 'ignore', 'ignore-user', 'redirect'
```

Comment:
Where do ad-attribution and ignore-user come from?

Reply:
I think 'ignore-user' action is one that the extension code sets itself, but wouldn't be found in the block list. I considered not including it here, we might not need it. It seemed less risky to include it though in case there's a code-path I missed - perhaps the extension mutates the block list to change the rule action to 'ignore-user' sometimes?

I included 'ad-attribution' as it's included in the `ActionName` type above. (See https://github.com/duckduckgo/privacy-grade/blob/main/src/classes/trackers.js#L70 .) I don't know much about that one, but I think @jonathanKingston would know more.

Reply:
This class is still this in MV3 to know what trackers were blocked by DNR rules? If so then I think we would show CTL requests as not being blocked.

Reply:
Yea, good point. It's possible the MV2 code-path would report these CTL requests as being allowed for MV3 before we eventually add MV2 support for the new CTL rule actions. We might have to just accept that, but I will mark this as a draft for now while we think about the trade-offs.

Reply:
> Where do ad-attribution and ignore-user come from? 

So I looked at the logic more today. I don't think we need these other actions, they aren't in the block list but are instead returned in the tracker Object (by `getTrackerData`). For example, if the tracker rule has a surrogate property, the action is reported as 'redirect' while the rule itself is left alone.

> This class is still this in MV3 to know what trackers were blocked by DNR rules? If so then I think we would show CTL requests as not being blocked.

Once both the MV2 and MV3 extension are using these Click to Load rule actions for blocking/redirecting Click to Load requests we should be good. They should be reported as well for MV3 as other blocked/redirected requests, since the `onBeforeRequest` code-path is still being run on MV3 for now.